.release-please-manifest.json

```diff
@@ -0,0 +1,3 @@
+{
+  ".": "1.22.1"
```

Comment:
The login and admin ui versions are prefixed with `v` (unlike our other rocks), does it need to be included?

Reply:
I think that app versions don't include the `v` and rock version include the `v` (see the existing release-please PR https://github.com/canonical/identity-platform-admin-ui/pull/528)